### ISSUE TITLE
Shadow issue with some blocks

### ISSUE THREAD

**Issue by Herobrine2143** [sp614x/optifine#1980]
I have an **issue** with the shaders, **perhaps is the shader's problem** but I want to make sure, this is happening with a big portion of all my shaders pack but not all of them. I **used SEUS Renewed 1.0.0**
![2018-10-26_21 00 17](https://user-images.githubusercontent.com/22053132/47587400-0db8ba80-d963-11e8-800c-740ff2beb1d5.png)
![2018-10-26_21 00 25](https://user-images.githubusercontent.com/22053132/47587401-0e515100-d963-11e8-9272-40f808ec6145.png)
![2018-10-26_21 00 26](https://user-images.githubusercontent.com/22053132/47587402-0e515100-d963-11e8-9ea2-7c85ee250af3.png)
![2018-10-26_21 00 28](https://user-images.githubusercontent.com/22053132/47587403-0e515100-d963-11e8-9487-feade9d0ea18.png)
![2018-10-26_21 00 11](https://user-images.githubusercontent.com/22053132/47587405-0e515100-d963-11e8-8dd8-c365c3590f47.png)
![2018-10-26_21 00 13](https://user-images.githubusercontent.com/22053132/47587406-0e515100-d963-11e8-977c-df520b98567f.png)

I'm taking **screenshot with** the shaders and **without** them, you can notice the difference.

Thank you for your time.

Herobrine_2143

**Comment by realjackasterisk** [comment]
What version are you playing on? SEUS hasn't been updated for 1.13+ yet.

**Comment by sp614x** [comment]
SEUS and most other shader packs have to be updated to 1.13.

**Comment by Herobrine2143** [comment]
> 
> 
> SEUS and most other shader packs have to be updated to 1.13.

Yeah But this is also happening in 1.12 which is the same issue, or maybe is not, I didn't check, but this is what happens to me, but apparently changing also texture packs makes 'em not only shadowy but also brighter.
